### PR TITLE
Dockerfile - Bug fix for rocm docker build and deploy

### PR DESCRIPTION
**Pull request by yukirora**
**Description**
Bug fix for rocm docker build and deploy.

### CODE REVIEW

**Review by cp5555 — APPROVED** [review]

**Review by cp5555 — APPROVED** [review]

File: superbench/runner/playbooks/deploy.yaml [hunk 1]
```diff
@@ -100,7 +100,7 @@
         docker run -itd --name={{ container }} \
           --privileged --net=host --ipc=host \
           {{ '--gpus=all' if nvidia_gpu_exist else '' }} \
-          {{ '--security-opt seccomp=unconfined --group-add video' if amd_gpu_exist else '' }} \
+          {{ '--security-opt seccomp=unconfined --group-add video --device=/dev/kfd --device=/dev/dri  --cap-add=SYS_PTRACE  --shm-size=16G' if amd_gpu_exist else '' }} \
```
Comment (yzygitzh) [inline superbench/runner/playbooks/deploy.yaml:103]: Remove redundant spaces?

File: third_party/Makefile [hunk 2]
```diff
@@ -86,12 +86,12 @@ ifneq (,$(wildcard fio/Makefile))
...
 endif
 
-# Build rccl-tests from commit 2a18737 of default branch.
+# Build rccl-tests with develop branch.
 rocm_rccl_tests: sb_micro_path
-ifneq (, $(wildcard rccl-tests/Makefile))
-	cd ./rccl-tests && make MPI=1 MPI_HOME=$(MPI_HOME) HIP_HOME=$(HIP_HOME) RCCL_HOME=$(RCCL_HOME) -j
-	cp -v ./rccl-tests/build/* $(SB_MICRO_PATH)/bin/
-endif
+	if [ -d rccl-tests ]; then rm -rf rccl-tests ; fi
```
Comment (yzygitzh) [inline third_party/Makefile:91]: Why do we remove rccl-tests from third party? Shall we update rccl submodule's commit ID instead?

Comment (yukirora) [reply third_party/Makefile:91]: ok, revised

File: superbench/runner/playbooks/deploy.yaml (hunk 1 above)
Comment (yukirora) [reply superbench/runner/playbooks/deploy.yaml:103]: updated

**Review by yzygitzh — APPROVED** [review]

File: superbench/runner/playbooks/deploy.yaml [hunk 3]
```diff
@@ -100,7 +100,7 @@
         docker run -itd --name={{ container }} \
           --privileged --net=host --ipc=host \
           {{ '--gpus=all' if nvidia_gpu_exist else '' }} \
-          {{ '--security-opt seccomp=unconfined --group-add video' if amd_gpu_exist else '' }} \
+          {{ '--security-opt seccomp=unconfined --group-add video --device=/dev/kfd --device=/dev/dri --cap-add=SYS_PTRACE --shm-size=16G' if amd_gpu_exist else '' }} \
```
[the remark this answers is not in the document]
Comment (yukirora) [reply superbench/runner/playbooks/deploy.yaml:103]: this new flags are recommended by amd people

**Pull request merged**